README.md

```diff
@@ -1,32 +1,169 @@
+> 
+> Create a new yaml file `azure_openai.yaml` with following template in the `my_chatbot` folder. Replace the `api_key` and `api_base` with your own Azure OpenAI API key and endpoint:
+> 
+> ```yaml
+> $schema: https://azuremlschemas.azureedge.net/promptflow/latest/AzureOpenAIConnection.schema.json
+> name: azure_open_ai_connection # name of the connection
+> type: azure_open_ai  # Azure Open AI 
+> api_key: "<aoai-api-key>" # replace with your Azure OpenAI API key
+> api_base: "aoai-api-endpoint" # replace with your Azure OpenAI API endpoint
+> api_type: "azure" 
+> api_version: "2023-03-15-preview" # replace with your Azure OpenAI API version
+> ```
+> 
+> Establish the connection by running:
+> ```sh
+> pf connection create --file ./my_chatbot/azure_openai.yaml
+> ```
+> </details>
+
+For Open AI connection, directly establish the connection by running:
+
+
+```sh
+# Override keys with --set to avoid yaml file changes
+pf connection create --file ./my_chatbot/openai.yaml --set api_key=<your_api_key>
+```
+
+<details>
+<summary><b>Chat with your flow</b></summary>
+In the `my_chatbot` folder, there's a `flow.dag.yaml` file that outlines the flow, including inputs/outputs, tools, nodes, etc. Note we're using the connection named `open_ai_connection` in the `chat` node.
+</details>
+
+> <details>
+> <summary>For Azure Open AI users, modify this file accordingly.</summary>
+> Replace the 'node:' section with following content and specify the 'deployment_name' to the model deployment you'd like to use.
+>
+> ```yaml
+> nodes:
+> - name: chat
+>   type: llm
+>   source:
+>     type: code
+>     path: chat.jinja2
+>   inputs:
+>     deployment_name: <your_azure_open_ai_deployment_name>
+>     max_tokens: '256'
+>     temperature: '0.7'
+>     chat_history: ${inputs.chat_history}
+>     question: ${inputs.question}
+>   api: chat
+>   connection: azure_open_ai_connection
+> ```
+>  </details>
+
+Interact with your chatbot via (Press `Ctrl + C` to end the session.):
+
+```sh
+pf flow test --flow ./my_chatbot --interactive
+```
+
+Congratulations! You have now prototyped your chatbot! 🎉 What's next?
+
+<details>
+<summary><b>Ensuring "High Quality” in production scenarios</b></summary>
+
+LLMs' randomness can yield unstable answers. Fine-tuning prompts can improve output reliability.  For accurate quality assessment, it's essential to test with larger datasets and compare outcomes with the ground truth.
+
+During fine-tuning the prompt, we also consider to strike a balance between the accuracy and the token cost of the LLM.
+
+Invest just 15 minutes to understand how prompt flow accelerates prompt tuning, testing, and evaluation, to find an ideal prompt **(accuracy ↑,token ↓)**
+<img src="examples/tutorials/quick-start/media/realcase.png" alt="comparison resutl" width=80%>
+</details>
+
+Try the [15-mins Easy Case](examples/tutorials/quick-start/promptflow-quality-improvement.md) on Tuning ➕ Batch Testing ➕ Evaluation ➡ Quality ready for production.
```

Comment:
This looks like a tune promt using variant tutorial to me.
I think we should still recommend user go to Getting Started with Prompt Flow first. 
Then to this new tutorial and chat-with-pdf one.